app/services/concerns/iiif_thumbnail_paths.rb

```diff
@@ -22,8 +22,10 @@ def iiif_thumbnail_path(file_set, size)
       )
     end
 
-    def thumbnail?(_thumbnail)
-      true
+    def thumbnail?(thumbnail)
```

Comment:
Please use Yard to add method documentation especially about the type of `thumbnail`.